discovery-provider/src/utils/helpers.py

```diff
@@ -343,6 +343,29 @@ def create_track_route_id(title, handle):
 
     return f"{sanitized_handle}/{sanitized_title}"
 
+
+def create_track_slug(title, conflict_index=0):
```

Comment:
nit: would be nice to have just a one sentence doc explaining what format this puts slugs in